linux_os/guide/system/software/integrity/crypto/configure_crypto_policy/oval/fedora.xml

```diff
@@ -1,3 +1,34 @@
+{{%- macro crypto_policy_symlink_criterion(library) %}}
+      <criterion comment="Check if /etc/crypto-policies/back-ends/{{{ library }}}.config is linked to /usr/share/crypto-policies/POLICY/{{{ library }}}.txt" test_ref="test_crypto_policy_{{{ library }}}_symlink" />
+{{%- endmacro %}}
+
+{{%- macro crypto_policy_symlink_check(library) %}}
+  <local_variable id="var_crypto_policy_{{{ library }}}_path" datatype="string" comment="Regex variable for canonical path to itargeted {{{ library }}} policy" version="1">
+    <concat>
+      <literal_component>^/usr/share/crypto-policies/</literal_component>
+      <variable_component var_ref="var_system_crypto_policy"/>
+      <literal_component>/{{{ library }}}.txt$</literal_component>
+    </concat>
+  </local_variable>
+
+  <unix:symlink_test id="test_crypto_policy_{{{ library }}}_symlink"
+  comment="check if /etc/crypto-policies/back-ends/{{{ library }}}.config is linked to crypto-policies"
```

Comment:
Suggestion for comment: `check if /etc/crypto-policies/back-ends/{{{ library }}}.config is linked to correct target library config`